appmap/test/data/unittest/simple/test_simple.py

```diff
@@ -1,6 +1,10 @@
 import unittest
 from unittest.mock import patch
 
+# Importing from decouple will cause a failure if we're not hooking
+# finders correctly.
+from decouple import config
```

Comment:
What makes `decouple` special here?

Reply:
I'm not sure. It was in the stack trace provided by the user that ran into the problem. Its presence here reproduced the failure before I applied the fix